src/containers/redux/actions/jobActions.ts

```diff
@@ -1,18 +1,34 @@
 import pathStore from "../../apis/pathStore";
-import { ActionTypes, JobType } from "../contants/action-type";
+import { JobType } from "../contants/action-type";
 import { Dispatch } from "redux";
+import { FETCH_JOBS, SEARCH_JOBS } from "../reducers/jobReducer";
 
 //Asynchronous Function Redux Thunk
 //Fetch Jobs Data with FETCH_JOBS
 export const fetchJobs =
-  () =>
+  (itemSearch: string) =>
```

Comment:
Could we make this default is empty? Then we don't need to pass an empty string in `fetchJobs` functionality.